frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -17,9 +17,10 @@ type DetailsSectionProps = {
   title: string;
   isLoading: boolean;
   loadError?: Error;
-  isEmpty: boolean;
-  emptyState: React.ReactNode;
+  isEmpty?: boolean;
+  emptyState?: React.ReactNode;
```

Comment:
Was this change necessary? 🤔 I think it has value to always ask this question as the intent is for you to handle "empty" states...

Reply:
I changed the interface at first but then realized I needed them but forgot to change them back. This should be addressed in the follow-up fixes.

Reply:
Will be fixed in https://github.com/opendatahub-io/odh-dashboard/pull/2154